test/test_restful.py

```diff
@@ -276,6 +276,33 @@ def initthread():
     return True
 
 
+def launch_rust_cloudagent():
```

Comment:
Can we do a `@unittest.skipIf`? I do no think that is OK for a Python test to unconditionally run a test that depends on cargo and the Rust toolchain.

Reply:
Done! See above.